docs/open-source-project/release-notes/tizen-3-0-m2.md

```diff
@@ -662,38 +662,38 @@ As a common feature, multi-user support has been added.
     - Hardware integration has been upgraded.
     - Encrypted initial value support has been added.
     - Symmetric key (AES key) support has been added.
-    - All public APIs have been changed to non-privileged APIs.
-    - The following APIs have been added:
-      - APIs to handle PKCS12 files, such as `ckmc_save_pkcs12()` and `ckmc_get_pkcs12()`, have been added.
+    - All public API set have been changed to non-privileged API set.
+    - The following API set have been added:
+      - API set to handle PKCS12 files, such as `ckmc_save_pkcs12()` and `ckmc_get_pkcs12()`, have been added.
       - API for certificate verification with designated trust certificates has been added.
-      - API for OCSP (Online Certificate Status Protocol) has been added.
+      - API for Online Certificate Status Protocol (OCSP) has been added.
       - The `kmc_set_permission()` function to manage access control rules efficiently has been added.
       - The `ckmc_remove_alias()` function that deletes data in the database using an alias has been added.
       - Error code (`CKMC_ERROR_AUTHENTICATION_FAILED`) for per-row password mismatched error has been added.
       - API to create an AES key has been added.
-      - APIs for encryption and decryption have been added.
-      - APIs to handle cryptographic parameters have been added.
-    - The following APIs have been changed:
-      - Platform-level privilege has been removed from the key-manager control APIs.
+      - API set for encryption and decryption have been added.
+      - API set to handle cryptographic parameters have been added.
+    - The following API set have been changed:
+      - Platform-level privilege has been removed from the key-manager control API set.
       - Getting a certificate chain with the aliases API has been deprecated.
   - Web application encryption
     - Web application (WGT) encryption feature is moved from secure-storage (until Tizen 2.4) to libwebappenc (since Tizen 3.0).
-  - YACA (Yet Another Crypto API)
+  - Yet Another Crypto API (YACA)
     - A new crypto API package, YACA, has been introduced in Tizen 3.0.
-      - It provides the application with the following stable cryptographic APIs:
-        - APIs for encryption/decryption operations with symmetric keys and sealing and opening operations with asymmetric keys
-        - APIs for creating and verifying a signature, calculating HMAC/CMAC, and calculating a message digest
-        - APIs for key handling operations, such as generating, importing, and exporting a key, and deriving a key from a password
-        - Simple APIs for cryptographic operations
-        - APIs for low-level RSA operations
+      - It provides the application with the following stable cryptographic API set:
+        - API set for encryption/decryption operations with symmetric keys and sealing and opening operations with asymmetric keys.
```

Comment:
From line 684 to line 688, it looks like they are not "sentences".  If so, are they need period?

Reply:
I have removed